styles.css

```diff
@@ -9,6 +9,7 @@ body{
 }
 header, footer, aside, .main{
   padding: 1em;
+  margin: 0 auto;
 }
```

Comment:
not quite, we need to target all direct children of these elements, can you think of a css selector that would help you do that?